### PR TITLE
Try: G2-inspired iconography

### LINKED ISSUE #20284: Consolidating WordPress Icons
Related Issues: #9647, #17055 and #18667.

With the fresh introduction of `@wordpress/icons` we have a much better baseline from which to operate and distribute iconography. It's extensively covered in the different issues above that there's currently a mix of icons between the historical dashicons, specific custom icons created during the editor project cycle, and the use of some Material design icons either directly or inspired by. These have been done as necessity dictated. @jasmussen summarizes some of the design issues at play [in this comment](https://github.com/WordPress/gutenberg/issues/9647#issuecomment-456738147). Outside of those, there have also been concerns with licenses and visual incompatibilities to consider when considering Material icons.

So the `@wordpress/icons` package is a stepping stone towards achieving an ever-growing set of open source & well-designed icons for WordPress that takes advantage of larger icons, better legibility, and a scaling grid for mobile contexts.

In the process of exploring the design update of #18667 @pablohoneyhoney began to update some of the iconography in use (starting with block icons) with these principles in mind, and the results (from my perspective) have been wonderful. There has also been a proliferation of icons in dropdown menus that is unnecessary in places, prompting an opportunity to reevaluate these more coherently. The overall visual update also favors higher contrast colors in borders and text instead of subdued grays, so tweaking the icons to favor this presentation helps a lot.

That work has been happening on the [figma files](https://www.figma.com/file/fnyj380i05vGzuuH60frLQ/G2-Design) if you want to check it out and comment. (Also hopefully much easier to contribute there for designers.)

image

It's important to consolidate on a highly reusable icon component for block and plugin developers in the `wordpress/icons` package as that would allow us to control the entry point and swap out specific implementations of individual icons gradually as time and effort permits. There are also some early questions about how it could or whether it should propagate with some of the component set to other places of the admin interface, for which this coalescing would be helpful.

image

### PR DESCRIPTION
**Pull request by jasmussen**
closes #20284

This PR explores adding the new icons that are discussed in #20284.

_Note that many of these icons are still receiving feedback in the Figma file, and this PR will be updated to address any corrections made._

Screenshots:

Screenshot 2020-02-26 at 13 34 49

Screenshot 2020-02-26 at 13 35 01

Screenshot 2020-02-26 at 13 35 18

Screenshot 2020-02-26 at 13 35 43

Screenshot 2020-02-26 at 13 35 56

This branch is still in progress, as these new icons require a bit more work than just replacing SVG files. For instance, the dropdown indicator for more inline tools is currently drawn with CSS, and should be a chevron instead, and some icons are currently scaled down, so the CSS needs to be tuned.

However as a work in progress, I invite you to try out the current state of the branch. The immediate feeling is that the coherence feels very much elevated, as the icons are balanced towards the same G2 visual, causing a nice relationship between the dark stroke outlines and the reduced icons.

### CODE REVIEW

File: packages/icons/src/library/more-vertical.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+ */
+import { SVG, Path } from '@wordpress/primitives';
+
+const moreVertical = (
+	<SVG xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24">
+		<Path d="M13 19h-2v-2h2v2zm0-6h-2v-2h2v2zm0-6h-2V5h2v2z" />
+	</SVG>
+);
+
+export default moreVertical;
```
Comment (youknowriad) [inline packages/icons/src/library/more-vertical.js]: This should also be exposed in `icons/src/index.js`

File: packages/icons/src/index.js [hunk 2]
```diff
@@ -70,6 +70,7 @@ export { default as menu } from './library/menu';
 export { default as minus } from './library/minus';
 export { default as more } from './library/more';
 export { default as moreHorizontal } from './library/more-horizontal';
+export { default as moreVertifal } from './library/more-vertical';
```
Comment (youknowriad) [inline packages/icons/src/index.js]: This is not used in the UI yet right?

Comment (jasmussen) [reply packages/icons/src/index.js]: No, but when I get a chance I will. The top menu icon is vertical and is rotated with css, I'll remove the CSS and use the correct icon.

Comment (mtias) [reply packages/icons/src/index.js]: Also, typo :)

Comment (jasmussen) [reply packages/icons/src/index.js]: Fixed, that was embarrassing :D

**Comment by jasmussen** [comment]
I'm slightly pausing adding additional icons, both to let feedback be addressed in the Figma file, but also to look at improving the pipeline slightly. 

Currently the icons are sourced in Figma, and Figma has a good SVG exporter that makes batch exporting easy. But many of the icons are drawn using strokes instead of fills, and these strokes need to become fills in order to work properly as icons in the editor. 

https://github.com/elrumordelaluz/outline-stroke looks like it can help automate the conversion of strokes to fills. Combined with svgmin, the pipeline can be automated entirely.

**Comment by youknowriad** [comment]
The inserter icon doesn't work well if the button is not bordered or with a background. Maybe it's time to add back to the isPrimary or isSecondary variation for the appender and inbetween-inserter.

Also the ButtonAppender is still using the old icon.

**Comment by youknowriad** [comment]
I don't know if it's just a visual impression but the main inserter feels off-centered for me.

**Comment by jasmussen** [comment]
> The inserter icon doesn't work well if the button is not bordered or with a background. Maybe it's time to add back to the isPrimary or isSecondary variation for the appender and inbetween-inserter.

Oh that's right, I meant to mention that. This is something we'll have to solve, and I have a few ideas.

**Comment by jasmussen** [comment]
> I don't know if it's just a visual impression but the main inserter feels off-centered for me.

Good catch, I believe that should be fixed now.

**Comment by jasmussen** [comment]
I'm sad to note that the `outline-stroke` package is not useful for us:

Screenshot 2020-03-02 at 09 32 38

So instead I'll be looking at other ways to streamline in the near future.

**Comment by jasmussen** [comment]
Fixed this one, which has annoyed me for a while:

this one

**Comment by jasmussen** [comment]
Chevron icon for more inline tools:

Screenshot 2020-03-03 at 11 31 10

**Comment by jasmussen** [comment]
Retired the dropdown mixin, which also meant removing it from here:

Screenshot 2020-03-03 at 11 32 29

Better, even though it can become better still:

Screenshot 2020-03-03 at 11 35 58

**Comment by jasmussen** [comment]
Polished the sibling inserter. Before:

Screenshot 2020-03-03 at 11 37 58

It was inspired by this:

Screenshot 2020-03-03 at 11 41 50

After:

Screenshot 2020-03-03 at 12 24 45

I also added an animation-only delay:

delayed

**Comment by jasmussen** [comment]
Polished the appenders used in Buttons, Social Links, Navigation:

Screenshot 2020-03-03 at 12 55 23

**Comment by jasmussen** [comment]
Alright, pushed a bunch of polish today, and added the remaining icons.

Although there is already a great deal of feedback awaiting us in the Figma file to deal with, as a first stab at new icons, this PR is ready for final review. Any and all feedback can either be addressed directly here, or even in subsequent PRs. But as an initial pass, this PR leverages the new icon component and simplifies drastically the CSS necessary to show icons across the board, including a refactor of how the inline version of the appender works. 

The columns block is a good little microcosm of the polish:

columns

Lay your thoughts on me.

**Comment by jasmussen** [comment]
As far as automation goes, this is rife for automation, but for this first pass I did some of it somewhat manually. Because I did not find a good "stroke to outlines" system, I created an Illustrator action that outlined and compacted/flattened the icon. To be fair, this isn't something we've had automated in the past either, we've simply required in the README that the SVG input file was already outlined and flattened. 

SVGO on the commandline is a great way to simplify things. `svgo *.svg` does it. This was part of existing build processes, including the last step that also outputs React friendly .js files. 

Mostly storing here as a note to self to revisit later.

**Comment by shaunandrews** [comment]
Overall, this is really nice and it starts to bring the recent UI updates together in a more cohesive language. The thin lines of the new icons, focus states, and updated inserters all help to remove some of the _heft_ of the Gutenberg UI — very nice work.

--

image

The + icon appears to _not_ be centered. In order to actually center it within the 32px shape, it would need to be an a half-pixel position, since the width of the line is a single pixel. Would this lead to blurry lines? Is there another way to center the icon?

image

I'm seeing the inline inserter on an empty paragraph block with an "invisible" icon. When I hover, I see the icon so maybe there's some CSS mixup here.

image

The device dropdown is using different checkmarks than the Tools menu — I'd expect them to be the same. Also, the external icon here seems out of place now.

image

I _think_ this hover state is new in this PR (correct me if I'm wrong) and I'm not a fan. The border feels cluttered with the border on the bottom toolbar, and the arrows between the breadcrumbs.

**Comment by shaunandrews** [comment]
image

Last minute thought: Would it make sense to change the color of the icons in the inserter to match the darker color of the icons when shown in the block toolbar?

**Comment by jasmussen** [comment]
Excellent thoughts all around 👌 I'll address all of it.

**Comment by chrisvanpatten** [comment]
Is that quote icon from the screenshot above new? The bottom line (the hook? descender curly thing?) looks a touch long. Love the overall look though, and I like that it still feels like it has inherited something from Material, so there is a bit of consistency.

**Comment by jasmussen** [comment]
> Is that quote icon from the screenshot above new? The bottom line (the hook? descender curly thing?) looks a touch long

It shouldn't be. Although it's not shown directly in the issue, in #20284, the Figma file linked is the file I took it from. Your feedback is very valid, and Pablo who has been working on some of the icons has been looking at the feedback lately and I think will do a pass as soon as time permits, so if you'd be so kind as to add the comment directly to the figma file: https://www.figma.com/file/fnyj380i05vGzuuH60frLQ/G2-Design

Otherwise let me know and I'll write it there. On the master icon thread is also good.

> Love the overall look though, and I like that it still feels like it has inherited something from Material, so there is a bit of consistency.

Agree — it's so important to balance out the complexity of the UI, and while we've made strides in reducing and simplifying UI, the coherence and simplicity of the top toolbar icons feels like it takes that even further, in a way that only really seems possible if we design those to work together. 

Notably the alignment icons feel like they are compatible with Material as an expanded set, and I'm happy about the diversity we can bring.

**Comment by jasmussen** [comment]
Fixed the plus icon. 1x on 2x screens:

Screenshot 2020-03-04 at 10 12 43
Screenshot 2020-03-04 at 10 12 49

Zoomed pixels, 2x screens:

Screenshot 2020-03-04 at 10 13 58

1x screens:

Screenshot 2020-03-04 at 10 14 06

**Comment by jasmussen** [comment]
> I think this hover state is new in this PR (correct me if I'm wrong) and I'm not a fan. The border feels cluttered with the border on the bottom toolbar, and the arrows between the breadcrumbs.

Not an issue of this branch, but I fixed it.

**Comment by jasmussen** [comment]
I polished menus and checkmarks:

Screenshot 2020-03-04 at 10 46 10

Screenshot 2020-03-04 at 11 07 49

Note that the G2 designs put the icons to the right. This is a larger undertaking as it also requires taking good care of the keyboard shortcut, so I omitted that task for now, postponed to revisit later. That will also address the very large left padding on menu items. 

The external link icon is also something to either update, or discuss whether we can do without (the phrasing seems verbose enough).

That should address remaining feedback!

**Comment by jasmussen** [comment]
> I'm seeing the inline inserter on an empty paragraph block with an "invisible" icon. When I hover, I see the icon so maybe there's some CSS mixup here.

Sorry, forgot to respond to this one. 

This is unfortunately a case of TwentyTwenty overreaching slightly, and intentionally recoloring this button in the editor style. This is something we can report to trac if/when this lands.

File: packages/block-editor/src/components/block-list-appender/style.scss [hunk 3]
```diff
@@ -1,22 +1,22 @@
...
 	}
 }
 
 .block-list-appender.is-drop-target > div::before {
 	content: "";
 	position: absolute;
-	right: -$block-padding;
-	left: -$block-padding;
-	top: -$block-padding;
-	bottom: -$block-padding;
```
Comment (youknowriad) [inline packages/block-editor/src/components/block-list-appender/style.scss]: I wonder if this variable is still needed at all?

File: packages/block-editor/src/components/block-navigation/style.scss [hunk 4]
```diff
@@ -16,24 +16,6 @@ $tree-item-height: 36px;
 	margin: 0;
 }
 
-.block-editor-block-navigation__list .block-editor-button-block-appender {
```
Comment (youknowriad) [inline packages/block-editor/src/components/block-navigation/style.scss]: Is this not needed at all?

File: packages/block-editor/src/components/block-vertical-alignment-toolbar/icons.js [hunk 5]
```diff
@@ -4,37 +4,19 @@
...
-		xmlns="http://www.w3.org/2000/svg"
-		width="20"
-		height="20"
-		viewBox="0 0 24 24"
-	>
-		<Path fill="none" d="M0 0h24v24H0V0z" />
-		<Path d="M8 11h3v10h2V11h3l-4-4-4 4zM4 3v2h16V3H4z" />
+	<SVG xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24">
+		<Path d="M9 20h6V9H9v11zM4 4v1.5h16V4H4z" />
 	</SVG>
```
Comment (youknowriad) [inline packages/block-editor/src/components/block-vertical-alignment-toolbar/icons.js]: Should we move these to the icons package?

File: packages/block-editor/src/components/tool-selector/index.js [hunk 6]
```diff
@@ -16,22 +16,21 @@ import { useViewportMatch } from '@wordpress/compose';
...
-		width="20"
-		height="20"
+		width="24"
+		height="24"
 		viewBox="0 0 24 24"
 	>
-		<Path d="M6.5 1v21.5l6-6.5H21L6.5 1zm5.1 13l-3.1 3.4V5.9l7.8 8.1h-4.7z" />
+		<Path d="M9.4 20.5L5.2 3.8l14.6 9-2 .3c-.2 0-.4.1-.7.1-.9.2-1.6.3-2.2.5-.8.3-1.4.5-1.8.8-.4.3-.8.8-1.3 1.5-.4.5-.8 1.2-1.2 2l-.3.6-.9 1.9zM7.6 7.1l2.4 9.3c.2-.4.5-.8.7-1.1.6-.8 1.1-1.4 1.6-1.8.5-.4 1.3-.8 2.2-1.1l1.2-.3-8.1-5z" />
 	</SVG>
 );
```
Comment (youknowriad) [inline packages/block-editor/src/components/tool-selector/index.js]: Do we want to move these into the icons package?

File: packages/block-library/src/code/icon.js [hunk 7]
```diff
@@ -5,7 +5,6 @@ import { Path, SVG } from '@wordpress/components';
 
 export default (
 	<SVG viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg">
-		<Path d="M0,0h24v24H0V0z" fill="none" />
-		<Path d="M9.4,16.6L4.8,12l4.6-4.6L8,6l-6,6l6,6L9.4,16.6z M14.6,16.6l4.6-4.6l-4.6-4.6L16,6l6,6l-6,6L14.6,16.6z" />
+		<Path d="M14.5 6.4l-1 1.1 5.4 4.9-5.4 5 1 1.1 6.6-6.1-6.6-6zm-5 0l-6.6 6.1 6.6 6 1-1.1-5.4-4.9 5.4-5-1-1.1z" />
 	</SVG>
```
Comment (youknowriad) [inline packages/block-library/src/code/icon.js]: Weird, I thought I already moved this to the icons package?

File: packages/block-library/src/embed/icons.js [hunk 8]
```diff
@@ -1,31 +1,26 @@
...
-		<Path d="M21,4H3C1.9,4,1,4.9,1,6v12c0,1.1,0.9,2,2,2h18c1.1,0,2-0.9,2-2V6C23,4.9,22.1,4,21,4z M21,18H3V6h18V18z" />
-		<Polygon points="14.5 11 11 15.51 8.5 12.5 5 17 19 17" />
+		<Path d="M19 3H5c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2zM9.2 4.5H19c.3 0 .5.2.5.5v8.4l-3-2.9c-.3-.3-.8-.3-1 0L11.9 14 9 12c-.3-.2-.6-.2-.8 0l-3.6 2.6V9.8l4.6-5.3zm9.8 15H5c-.3 0-.5-.2-.5-.5v-2.4l4.1-3 3 1.9c.3.2.7.2.9-.1L16 12l3.5 3.4V19c0 .3-.2.5-.5.5z" />
 	</SVG>
 );
 export const embedVideoIcon = (
 	<SVG viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg">
-		<Path d="M0,0h24v24H0V0z" fill="none" />
-		<Path d="m10 8v8l5-4-5-4zm9-5h-14c-1.1 0-2 0.9-2 2v14c0 1.1 0.9 2 2 2h14c1.1 0 2-0.9 2-2v-14c0-1.1-0.9-2-2-2zm0 16h-14v-14h14v14z" />
+		<Path d="M19 3H5c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2zm.5 16c0 .3-.2.5-.5.5H5c-.3 0-.5-.2-.5-.5V9.8l4.7-5.3H19c.3 0 .5.2.5.5v14zM10 15l5-3-5-3v6z" />
```
Comment (youknowriad) [inline packages/block-library/src/embed/icons.js]: What about moving these to the icons package?

File: packages/block-library/src/html/icon.js [hunk 9]
```diff
@@ -5,6 +5,6 @@ import { SVG, Path } from '@wordpress/components';
 
 export default (
 	<SVG viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg">
-		<Path d="M4.5,11h-2V9H1v6h1.5v-2.5h2V15H6V9H4.5V11z M7,10.5h1.5V15H10v-4.5h1.5V9H7V10.5z M14.5,10l-1-1H12v6h1.5v-3.9  l1,1l1-1V15H17V9h-1.5L14.5,10z M19.5,13.5V9H18v6h5v-1.5H19.5z" />
+		<Path d="M4.5 11.4H1.8V9H.7v6h1.1v-2.6h2.8V15h1.1V9H4.5v2.4zM6.4 10h1.7v5h1.1v-5h1.7V9H6.4v1zm9.5-1l-1.5 2.7L13 9h-.9l-.8 6h1.1l.5-4 1.5 2.8L16 11l.5 4h1.1l-.8-6h-.9zm3.8 5V9h-1.1v6h3.6v-1h-2.5z" />
```
Comment (youknowriad) [inline packages/block-library/src/html/icon.js]: Same here, I thought I already moved this to the icons package?

File: packages/block-library/src/latest-posts/icon.js [hunk 10]
```diff
@@ -1,17 +1,10 @@
...
 	<SVG viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg">
-		<Path d="M0,0h24v24H0V0z" fill="none" />
-		<Rect x="11" y="7" width="6" height="2" />
-		<Rect x="11" y="11" width="6" height="2" />
-		<Rect x="11" y="15" width="6" height="2" />
-		<Rect x="7" y="7" width="2" height="2" />
-		<Rect x="7" y="11" width="2" height="2" />
-		<Rect x="7" y="15" width="2" height="2" />
-		<Path d="M20.1,3H3.9C3.4,3,3,3.4,3,3.9v16.2C3,20.5,3.4,21,3.9,21h16.2c0.4,0,0.9-0.5,0.9-0.9V3.9C21,3.4,20.5,3,20.1,3z M19,19H5V5h14V19z" />
+		<Path d="M18 4H6c-1.1 0-2 .9-2 2v12c0 1.1.9 2 2 2h12c1.1 0 2-.9 2-2V6c0-1.1-.9-2-2-2zm.5 14c0 .3-.2.5-.5.5H6c-.3 0-.5-.2-.5-.5V6c0-.3.2-.5.5-.5h12c.3 0 .5.2.5.5v12zM7 11h2V9H7v2zm0 4h2v-2H7v2zm3-4h7V9h-7v2zm0 4h7v-2h-7v2z" />
```
Comment (youknowriad) [inline packages/block-library/src/latest-posts/icon.js]: Same, I thought I already moved this to the icons package? I believe this file is not used at all and should be removed.

File: packages/block-library/src/code/icon.js (hunk 7 above)
Comment (youknowriad) [reply packages/block-library/src/code/icon.js]: Maybe this file is not used anymore and should just get removed.

File: packages/block-library/src/navigation-link/icons.js [hunk 11]
```diff
@@ -5,8 +5,7 @@ import { Path, SVG } from '@wordpress/components';
 
 export const ToolbarSubmenuIcon = () => (
 	<SVG xmlns="http://www.w3.org/2000/svg" width="24" height="24">
-		<Path d="M14 5h8v2h-8zm0 5.5h8v2h-8zm0 5.5h8v2h-8zM2 11.5C2 15.08 4.92 18 8.5 18H9v2l3-3-3-3v2h-.5C6.02 16 4 13.98 4 11.5S6.02 7 8.5 7H12V5H8.5C4.92 5 2 7.92 2 11.5z" />
-		<Path fill="none" d="M0 0h24v24H0z" />
+		<Path d="M2 12c0 3.6 2.4 5.5 6 5.5h.5V19l3-2.5-3-2.5v2H8c-2.5 0-4.5-1.5-4.5-4s2-4.5 4.5-4.5h3.5V6H8c-3.6 0-6 2.4-6 6zm19.5-1h-8v1.5h8V11zm0 5h-8v1.5h8V16zm0-10h-8v1.5h8V6z" />
```
Comment (youknowriad) [inline packages/block-library/src/navigation-link/icons.js]: Should we move this to the icons package.

File: packages/block-library/src/navigation/use-block-navigator.js [hunk 12]
```diff
@@ -14,10 +14,10 @@ const NavigatorIcon = (
 	<SVG
 		xmlns="http://www.w3.org/2000/svg"
 		viewBox="0 0 24 24"
-		width="20"
-		height="20"
+		width="24"
+		height="24"
 	>
-		<Path d="M5 5H3v2h2V5zm3 8h11v-2H8v2zm9-8H6v2h11V5zM7 11H5v2h2v-2zm0 8h2v-2H7v2zm3-2v2h11v-2H10z" />
+		<Path d="M13.8 5.2H3v1.5h10.8V5.2zm-3.6 12v1.5H21v-1.5H10.2zm7.2-6H6.6v1.5h10.8v-1.5z" />
```
Comment (youknowriad) [inline packages/block-library/src/navigation/use-block-navigator.js]: Icons package?

File: packages/edit-post/src/components/preview-options/index.js [hunk 13]
```diff
@@ -7,22 +7,22 @@ import {
...
+	Path,
 	SVG,
 } from '@wordpress/components';
 import { useSelect, useDispatch } from '@wordpress/data';
 import { PostPreviewButton } from '@wordpress/editor';
 import { __, _x } from '@wordpress/i18n';
-import { external } from '@wordpress/icons';
+import { external, check } from '@wordpress/icons';
 
 const downArrow = (
```
Comment (youknowriad) [inline packages/edit-post/src/components/preview-options/index.js]: Isn't this already on the icons package?

**Comment by jasmussen** [comment]
You're a treasure, Riad. I don't know how you manage to review so thoroughly, and create new PRs at the same time. Thank you.

File: packages/block-editor/src/components/block-list-appender/style.scss (hunk 3 above)
Comment (jasmussen) [reply packages/block-editor/src/components/block-list-appender/style.scss]: Screenshot 2020-03-04 at 14 20 01

It is. 

I recall us recently removing a min-height on the "fake appender", btw. That makes it 22px tall, when it should be 28px tall.

File: packages/block-editor/src/components/block-navigation/style.scss (hunk 4 above)
Comment (jasmussen) [reply packages/block-editor/src/components/block-navigation/style.scss]: Some of it was, yes, good catch. Most of it wasn't, though.

Screenshot 2020-03-04 at 14 26 16

**Comment by shaunandrews** [comment]
Looking real good. Very nice work.

**Review by youknowriad — APPROVED** [review]
This is looking great for me 👍 
I think now it's best to ship and polish any details in dedicated PRs like moving the specific icons to the icons package.

**Comment by jasmussen** [comment]
Riad you speedy speedy man! Did you go ahead and address all the "icon package" items you reported? Because I was going to! But I was going to address it :)

One note before you merge, though — I noticed that the checkbox form control is now looking a bit strained, I'd like to see what I can do with it before merge.

Screenshot 2020-03-05 at 09 28 16

**Comment by jasmussen** [comment]
Polished checkboxes. Small screens:

Screenshot 2020-03-05 at 09 48 28

Big screens:

Screenshot 2020-03-05 at 09 49 10

**Comment by mapk** [comment]
🎉 Really well done! Thanks for tying together all the things!

**Comment by JohnRDOrazio** [comment]
Would it be possible to add the color-picker dashicon to gutenberg? I started porting from https://github.com/WordPress/dashicons/blob/master/svg-min/color-picker.svg like this:
https://github.com/JohnRDOrazio/gutenberg/blob/add-color-picker-icon/packages/icons/src/library/color-picker.js
But I'm not sure that's enough? Can this be looked into?

**Pull request merged**